playwright/test/view-items.test.ts

```diff
@@ -230,20 +211,14 @@ test('(15) | The location of the search results should be displayed', async ({
     }
   );
   await itemWithSearchAndStructuresAndQuery(context, page);
-
-  if (isMobile(page)) {
-    await page.getByRole('button', { name: 'Show info' }).click();
-  }
+  await accessSidebar(page);
 
   await page
     .getByRole('link')
     .filter({ hasText: 'Found on image 5 / 68' })
     .click();
 
-  if (!isMobile(page)) {
-    // we don't display this info on mobile as there is not enough room
```

Comment:
So from what I understand, it's not _visible_ but it is still rendered in the DOM, so this would also work for mobile. [Here is the playwright definition of visible](https://playwright.dev/docs/actionability#visible); I _think_ in this case it's because it's the parent that has `display:none` and not our locator itself that it still passes. 

If we think that test should not pass then we'd have to change the way we render it - in the meantime I think it's nice we test it on mobile too?

cc @gestchild @davidpmccormick